ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -527,3 +527,36 @@ if snpe_support_is_available
     include_directories: snpe_incdir
   )
 endif
+
+if nntrainer_support_is_available
+  filter_sub_nntrainer_sources = ['tensor_filter_nntrainer.cc']
+  nntrainer_inc=[]
+
+  if not get_option('enable-tizen')
+     nntrainer_inc += include_directories('../../../api/capi/include/platform')
```

Comment:
do we need this? I think this will be part of `nntrainer_dep` from the pkgconfig

Reply:
Well this is because ml-api-common.h which includes the tizen-error.h.